modules/widgets/twitter-timeline.php

```diff
@@ -307,7 +307,7 @@ public function update( $new_instance, $old_instance ) {
 	 */
 	public function get_docs_link( $hash = '' ) {
 		if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
-			$base_url = esc_url( Redirect::get_url( 'wpcom-support-widgets-twitter-timeline-widget' ) );
+			$base_url = 'https://support.wordpress.com/widgets/twitter-timeline-widget/';
```

Comment:
Maybe we can take that opportunity to update the link to its new URL?
https://wordpress.com/support/widgets/twitter-timeline-widget/

Reply:
Yeah, that's fine to do. I'll update it on the phab too after lunch.